content/docs/user-guide/testing/_index.md

```diff
@@ -4,7 +4,7 @@ date: 2021-03-02T00:23:56-05:00
 weight: 1300
 ---
 
-Test automation is the process of writing small snippets of code (tests) to verify the desired functionality of Open 3D Engine extensions and tools that you create.
+Test automation is the process of writing small snippets of code (tests) to verify the desired functionality of Open 3D Engine extensions and tools that you create.  Optimization is the process of making code perform and use resources in the most efficient way.
```

Comment:
What does the optimization blurb have to do with testing?
It doesn't seem relevant to this document

Reply:
The high level title/link to this document is `Testing and optimization` so I added a blurb about optimization - but I agree, it does seem like maybe I should just remove "optimization" from the title and make a new section entirely for optimization

Reply:
"pptimization" sounds a good folder to place the settings into